openslides_backend/action/actions/poll/vote.py

```diff
@@ -14,7 +14,7 @@
 from ..vote.user_token_helper import get_user_token
 
 
-@register_action("poll.vote")
+@register_action("poll.vote", internal=True)
```

Comment:
This is not possible, since the client can then no longer call this action and this is used in the staging server and full setup. Just overwrite the `check_permissions` method here to return directly, without any checks.

Reply:
...and create an issue

Reply:
Actually, an issue is not needed, since the action will be removed once #677 is done

Reply:
Rm `internal` and add an empty `check_permissions` to `poll.vote`.